requirements.txt

```diff
@@ -1,8 +1,5 @@
-keras
```

Comment:
requirements.txt seems to be inconsistent with setup.py

Reply:
Fixed in e0ad43d